### PR TITLE
Add links to stable docs

### PR DESCRIPTION
**Pull request by GeorgeWL**
note

Issue:

## How to test

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

**Comment by Hypnosphi** [comment]
Thanks @GeorgeWL 
Can you please add the same note for other addons as well?

File: addons/notes/README.md [hunk 1]
```diff
@@ -16,6 +15,7 @@ Storybook Addon Notes allows you to write notes (text or HTML) for your stories
 ![Storybook Addon Notes Demo](docs/demo.png)
 
 ### Getting Started
+**NOTE: Documentation on master branch is for alpha version, stable release is on [release/3.4](https://github.com/storybooks/storybook/tree/release/3.4/addons/)**
```
Comment (Hypnosphi) [inline addons/notes/README.md]: It would probably make sense to specify the particular addon in each link

Comment (ndelangen) [reply addons/notes/README.md]: Indeed, very very valuable 🙇

**Comment by ndelangen** [comment]
@GeorgeWL Do you want to apply the change the other addon readme files as well? 🙇

**Comment by danielduan** [comment]
I almost feel like we should have a tool for updating all the readmes. There's too many to do manually for every major or minor release.

**Comment by GeorgeWL** [comment]
I agree there

**Comment by Hypnosphi** [comment]
@danielduan this is a temporary measure, since 4.0 `master` will be stable again, and `next` will be default branch on GitHub for new releases instead

**Comment by Keraito** [comment]
That will definitely solve a lot of the issues that get opened from people confused about the docs.

**Review by ndelangen — APPROVED** [review]

**Pull request merged**